src/firebase.ts

```diff
@@ -24,4 +24,6 @@ const app = initializeApp(firebaseConfig);
 // init services
 export const auth = getAuth(app);
 
+export const db = getFirestore(app);
+
 export default getFirestore(app);
```

Comment:
this is doing the same thing, keep one and change where the other is being used, i think it's in the firebaeSnapshots file